### ISSUE TITLE
libaflfuzzer/fuzzbench abort

### ISSUE THREAD

**Issue by vanhauser-thc** [AFLplusplus/LibAFL#180]
to reproduce: `make test-run-libafl-libxslt_xpath` from AFLplusplus/fuzzbench

If the target is compiled with ubsan and it hits then the fuzzer dies:
```
['/out/xpath', '-x', '/out/xpath.dict', '/out/corpus', '/out/seeds']
Workdir: "/out"
[/libafl/libafl/src/events/simple.rs:314] "Spawning next client (id {})" = "Spawning next client (id {})"
[/libafl/libafl/src/events/simple.rs:314] ctr = 0
We're a client, let's fuzz :)
First run. Let's set it all up
Let's fuzz :)
UndefinedBehaviorSanitizer:DEADLYSIGNAL
==13==ERROR: UndefinedBehaviorSanitizer: SEGV on unknown address 0x000000000008 (pc 0x00000043240c bp 0x7ffe1d8d53c0 sp 0x7ffe1d8d40b0 T13)
==13==The signal is caused by a READ memory access.
==13==Hint: address points to the zero page.
    #0 0x43240c  (/out/xpath+0x43240c)
    #1 0x5c8fb5  (/out/xpath+0x5c8fb5)
    #2 0x5c5d80  (/out/xpath+0x5c5d80)
    #3 0x677a5f  (/out/xpath+0x677a5f)
    #4 0x7f2edc87983f  (/lib/x86_64-linux-gnu/libc.so.6+0x2083f)
    #5 0x40fa58  (/out/xpath+0x40fa58)

UndefinedBehaviorSanitizer can not provide additional info.
SUMMARY: UndefinedBehaviorSanitizer: SEGV (/out/xpath+0x43240c) 
==13==ABORTING
thread '<unnamed>' panicked at 'Fuzzer-respawner: Storing state in crashed fuzzer instance did not work, no point to spawn the next client! (Child exited with: 256)', /libafl/libafl/src/events/simple.rs:347:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by andreafioraldi** [comment]
`UndefinedBehaviorSanitizer:DEADLYSIGNAL` is the ubsan signal handler that should never appear... an error in inprocessexecutor?

**Comment by domenukk** [comment]
Fixed with ea40c21533acb3e825ad623172346007aa475139
